encoding/protobq/schema_test.go

```diff
@@ -0,0 +1,53 @@
+package protobq
+
+import (
+	"testing"
+
+	"cloud.google.com/go/bigquery"
+	"google.golang.org/genproto/googleapis/example/library/v1"
+	"google.golang.org/protobuf/proto"
+	"gotest.tools/v3/assert"
+)
+
+func TestSchemaOptions_InferSchema(t *testing.T) {
```

Comment:
Since proto messages are from `google.golang.org/genproto/googleapis` we could add a tests for a more complicated message. Thinking specifically of:
*  `oneof`
* repeated messages with `oneof`
* wrapper types

IIRC, `iotv1` has some pretty complex types.

Reply:
Agree - but now that I've set the repo up with protobuf stub generation, I'll be adding test cases with custom protos instead.

Will try to implement map support 100% - then take a closer look at `oneof`.